coremis_app_integration/endpoints/login.py

```diff
@@ -0,0 +1,59 @@
+from django.utils.translation import gettext_lazy as _
+from rest_framework.authentication import authenticate
+from rest_framework.decorators import (
+    authentication_classes,
+    permission_classes,
+    api_view
+)
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework import status
+
+from coremis_app_integration.authorization import (
+    CustomBasicAuthentication,
+    DETAILS,
+    FAILED,
+    SUCCESS
+)
+from payroll.models import PaymentPoint
+
+
+@api_view(['POST'])
+@authentication_classes([CustomBasicAuthentication])
```

Comment:
Is this needed when 
```
authenticated_user = authenticate(username=username, password=password)
```
is latter on in function body?

Reply:
Yes, to keep this strange specification from WorldBank team to have BASIC auth and POST in every endpoints. I can check this when I will be able to run application from provided APK